### DISCUSSION TITLE
New Data Analyst here!

### DISCUSSION THREAD

**Discussion by ahmed-shuja-msdeim-ned** [community/community#140660] [New to GitHub]
Hey Everyone! I am [Ahmed](https://www.linkedin.com/in/enggahmedshuja/), currently doing my Masters degree in Data Engineering & Information Management while working as a Data Analyst at [@qordata](https://www.qordata.com/). 
I am so glad to be a part of the GitHub community! You can connect with me over Power BI & Fabric discussions and queries :)

**Comment by A-Odunayo** [comment]
Welcome 👍🏼 

I am also new here as a Data Analyst in Training.

**Comment by ghost** [comment]
Hello Everybody!!
Shivam Shukla

**Comment by ghostinhershell** [comment]
Welcome to GitHub and the GitHub Community @ahmed-shuja-msdeim-ned 🎆 ! We're glad you're here and can't wait to see where your GitHub journey takes you. Check out this [check-in](https://github.com/orgs/community/discussions/42302#discussion-4683140) with links on how to choose your first project to contribute to, a blog with getting started resources, and a new user checklist.

Best of luck!
